pinot-core/src/main/java/org/apache/pinot/core/data/readers/RecordReaderFactory.java

```diff
@@ -46,10 +46,10 @@ public static RecordReader getRecordReader(SegmentGeneratorConfig segmentGenerat
     // If this is set, this will override the file format
     if (recordReaderPath != null) {
       if (fileFormat != FileFormat.OTHER) {
-        // We currently have default file format set to AVRO inside segment generator config,
-        // do not want to break this behavior for clients.
-        LOGGER.warn("Using recordReaderPath {} to read segment, ignoring fileformat {}",
-            recordReaderPath, fileFormat.toString());
+        // NOTE: we currently have default file format set to AVRO inside segment generator config, do not want to break
+        // this behavior for clients.
+        LOGGER
```

Comment:
Previous format looks better to me :)

Reply:
I agree, but just follow the auto-format...